File: leetcode-submissions/3110_Score_of_a_String.cpp
class Solution {
public:
    int scoreOfString(string s) {
        //now calculating the ascii value of the charcters and then adding it
        int ans=0;
        for(int i=0;i<s.size()-1;i++){
            int a =s[i]-'a';
            int b=s[i+1]-'a';
            ans+=abs(a-b);
        }
        return ans;
    }
};

File: leetcode-submissions/3110_Score_of_a_String.ts
function scoreOfString(s: string): number {
  let ans:number =0;
  for(let i:number =0;i<s.length-1;i++){
    let a : number =s.charCodeAt(i);
    let b:number =s.charCodeAt(i+1);
    ans+=Math.abs(a-b);

  } 
  return ans ; 
};